File: PlatziCodingChallenge_Dia3/Reloj.js
//Reloj en JavaScript
function timeInSeconds(Hours, Minutes) {
    return (Hours * 3600) + (Minutes * 60)
}

var horas = prompt('Introduzca las horas')
var minutos = prompt('Introduzca los minutos')
alert(`Tiempo en segundos: ${timeInSeconds(horas, minutos)}`)

File: Dia3/Reloj.py
#Reloj en Python
def timeInSeconds(hours, minutes):
    return hours*3600 + minutes*60

horas = int(input('Introduzca las horas: '))
minutos = int(input('Introduzca los minutos: '))
print('Tiempo en segundos: {}'.format(timeInSeconds(horas, minutos)))